### ISSUE TITLE
catkin_make error

### ISSUE THREAD

**Issue by xsmxsm** [ros-industrial-attic/fermi#36]
```
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp: In member function ‘void moveit_cartesian_plan_plugin::widgets::PathPlanningWidget::init()’:
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp:73:16: error: ‘class Ui::PathPlanningWidget’ has no member named ‘combo_planGroup’
    connect(ui_.combo_planGroup,SIGNAL(currentIndexChanged ( int )),this,SLOT(selectedPlanGroup(int)));
                ^
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp: In member function ‘void moveit_cartesian_plan_plugin::widgets::PathPlanningWidget::getCartPlanGroup(std::vector<std::basic_string<char> >)’:
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp:84:10: error: ‘class Ui::PathPlanningWidget’ has no member named ‘combo_planGroup’
      ui_.combo_planGroup->addItem(QString::fromStdString(group_names[i]));
          ^
make[2]: *** [fermi-indigo-devel/moveit_cartesian_plan_plugin/CMakeFiles/moveit_cartesian_plan_plugin.dir/src/widgets/path_planning_widget.cpp.o] 错误 1
make[1]: *** [fermi-indigo-devel/moveit_cartesian_plan_plugin/CMakeFiles/moveit_cartesian_plan_plugin.dir/all] 错误 2
make: *** [all] 错误 2
```

**Comment by rkojcev** [comment]
Hi @xsmxsm,

This seems to me its an UI problem, in the combo_planGroup. I have quickly rechecked this and the component combo_planGroup is in the UI. For me this compiles fine.

Can you tell me more details about your setup, system, and maybe Qt version? That way I can try to reproduce this

**Comment by xsmxsm** [comment]
Its ubuntu14.04 ros-indigo  ,I install QT through `sudo apt-get install...` version maybe 1.0.0

``` cmake
# generated from catkin/cmake/template/pkgConfig-version.cmake.in
set(PACKAGE_VERSION "1.0.0")

set(PACKAGE_VERSION_EXACT False)
set(PACKAGE_VERSION_COMPATIBLE False)

if("${PACKAGE_FIND_VERSION}" VERSION_EQUAL "${PACKAGE_VERSION}")
  set(PACKAGE_VERSION_EXACT True)
  set(PACKAGE_VERSION_COMPATIBLE True)
endif()

if("${PACKAGE_FIND_VERSION}" VERSION_LESS "${PACKAGE_VERSION}")
  set(PACKAGE_VERSION_COMPATIBLE True)
endif()
```

**Comment by rkojcev** [comment]
can you go to the catkin workspace and try running:

$ rosdep install --from-paths src --ignore-src --rosdistro indigo -y

**Comment by xsmxsm** [comment]
xsm@xsm-23-g131cn:~/catkin_ws1$ rosdep install --from-paths src --ignore-src --rosdistro indigo -y
WARNING: Package "ompl" does not follow the version conventions. It should not contain leading zeros (unless the number is 0).
ERROR: the following packages/stacks could not have their rosdep keys resolved
to system dependencies:
robotiq_cpp_control: Cannot locate rosdep definition for [obj_interface]
fermi_irb2400_moveit_config: Cannot locate rosdep definition for [abb_common]

**Comment by xsmxsm** [comment]
Do I need to change the file fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp ?

**Comment by xsmxsm** [comment]
xsm@xsm-23-g131cn:~/catkin_ws1$ rosdep install --from-paths src --ignore-src --rosdistro indigo -y
WARNING: Package "ompl" does not follow the version conventions. It should not contain leading zeros (unless the number is 0).
#All required rosdeps installed successfully

Its OK,but the error is still have.
[ 83%] Built target abb_robot_state
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp: In member function ‘void moveit_cartesian_plan_plugin::widgets::PathPlanningWidget::init()’:
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp:73:19: error: ‘class Ui::PathPlanningWidget’ has no member named ‘combo_planGroup’
       connect(ui_.combo_planGroup,SIGNAL(currentIndexChanged ( int )),this,SLOT(selectedPlanGroup(int)));
                   ^
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp: In member function ‘void moveit_cartesian_plan_plugin::widgets::PathPlanningWidget::getCartPlanGroup(std::vectorstd::basic_string<char >)’:
/home/xsm/catkin_ws1/src/fermi-indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.cpp:84:10: error: ‘class Ui::PathPlanningWidget’ has no member named ‘combo_planGroup’
      ui_.combo_planGroup->addItem(QString::fromStdString(group_names[i]));
          ^
make[2]: **\* [fermi-indigo-devel/moveit_cartesian_plan_plugin/CMakeFiles/moveit_cartesian_plan_plugin.dir/src/widgets/path_planning_widget.cpp.o] 错误 1
make[1]: **\* [fermi-indigo-devel/moveit_cartesian_plan_plugin/CMakeFiles/moveit_cartesian_plan_plugin.dir/all] 错误 2
make: **\* [all] 错误 2
Invoking "make -j4 -l4" failed

**Comment by xsmxsm** [comment]
catkin_make  success.
but when I use this plugin in rviz , there is still a problem ,I can't add point in it.

[ INFO] [1464247424.826011817]: Ready to take MoveGroup commands for group manipulator.
Object::connect: No such slot moveit_cartesian_plan_plugin::widgets::PathPlanningWidget::selectedPlanGroup(int)
Object::connect:  (sender name:   'combo_planGroup')
Object::connect:  (receiver name: 'PathPlanningWidget')
[ INFO] [1464247425.835644577]: initializing..
Object::connect: No such signal GenerateCartesianPath::sendCartPlanGroup(std::vector< std::string >)
Object::connect:  (receiver name: 'PathPlanningWidget')
Object::connect: No such signal moveit_cartesian_plan_plugin::widgets::PathPlanningWidget::sendSendSelectedPlanGroup(int)
Object::connect:  (sender name:   'PathPlanningWidget')
Object::connect: No such slot GenerateCartesianPath::initRvizDone()
Object::connect:  (sender name:   'CartesianPathPlannerPlugin')
[ INFO] [1464247425.835863144]: ready.

**Comment by rkojcev** [comment]
as pointed out by your error messages seems that there is a problem in the UI and that is missing a component. I have checked and this component exists and compiles fine (I have tried with clean compilation). Did you tried to modify the UI in any way? Did you clone the indigo-devel branch?

If you look into the indigo-devel branch of this repository the missing components you are reporting are actually in the code
https://github.com/ros-industrial-consortium/fermi/blob/indigo-devel/moveit_cartesian_plan_plugin/src/widgets/path_planning_widget.ui#L137
